robot/rospackages/src/mcu_control/scripts/CommandParser.py

```diff
@@ -26,4 +27,16 @@ def get_command(command_name, deviceToSendTo):
         if command_name == out_command[0]:
             return out_command
 
-    return None
\ No newline at end of file
+    return None
+
+
+class EmptyMock(object):
```

Comment:
This is a very smart solution, good stuff

Reply:
yeah I had seen in a talk a mention of ruby's method missing, found out python has a similar thing